astropy/constants/tests/test_prior_version.py

```diff
@@ -0,0 +1,157 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+# TEST_UNICODE_LITERALS
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from ...extern import six
+
+import copy
+
+import pytest
+
+from .. import Constant
+from ...units import Quantity as Q
+
+
+def test_c():
+
+    from ..CODATA2010 import c
+
+    # c is an exactly defined constant, so it shouldn't be changing
+    assert c.value == 2.99792458e8  # default is S.I.
+    assert c.si.value == 2.99792458e8
+    assert c.cgs.value == 2.99792458e10
+
+    # make sure it has the necessary attributes and they're not blank
+    assert c.uncertainty == 0  # c is a *defined* quantity
+    assert c.name
+    assert c.reference
+    assert c.unit
+
+
+def test_h():
+
+    from ..CODATA2010 import h
+
+    # check that the value is fairly close to what it should be (not exactly
+    # checking because this might get updated in the future)
```

Comment:
This should be exact, right, since `CODATA2010` can *not* be updated in the future! In fact, you might to explicitly check that it is *not* identical to `CODATA2014` (assuming that is the case)